client/src/components/Links/InviteLink.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { useTheme } from "@material-ui/core/styles";
+import * as S from "./style";
+import * as T from "../Typograpy";
+import Copy from "../Copy";
+
+const InviteLink = ({ color, url, background }) => {
+  const theme = useTheme();
+
+  return (
+    <S.InviteLinkWrapper background={background}>
+      <S.InviteLink to={url} target="_blank" color={color}>
+        <T.Body14B
+          color={color || theme.colors.pink}
```

Comment:
I think it would be better if we provide the color as the name of the color not the value
```suggestion
          color={heme.colors[color] || theme.colors.pink}
```
so no need to use `useTheme` in the parent component

Reply:
done